### PR TITLE
Rename CosineScheduler to CosineAnnealingLR

### PR DESCRIPTION
**Pull request by duchengyao**
image

### CODE REVIEW

**Comment by duchengyao** [comment]
与 oneflow 对齐 [tf.keras]( https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/CosineDecay?hl=zh-cn) 的 [CosineScheduler](https://oneflow.readthedocs.io/en/master/optim.html?highlight=cosines#oneflow.experimental.optim.lr_scheduler.CosineScheduler) 出自同一篇论文[SGDR: Stochastic Gradient Descent with Warm Restarts](https://arxiv.org/abs/1608.03983)，关闭PR

**Comment by wyg1997** [comment]
这个PR把之前的 CosineScheduler 改成Pytorch的命名吧，不用重新写了。我们eager模式还是对齐Pytorch的接口，以后会逐步把lazy的风格也迁移过来。

**Comment by duchengyao** [comment]
> 这个PR把之前的 CosineScheduler 改成Pytorch的命名吧，不用重新写了。我们eager模式还是对齐Pytorch的接口，以后会逐步把lazy的风格也迁移过来。

好的，我来做

**Comment by wyg1997** [comment]
这个Scheduler的文件名也需要改一下

**Comment by duchengyao** [comment]
> 这个Scheduler的文件名也需要改一下

Resolved

**Review by wyg1997 — APPROVED** [review]

**Review by wyg1997 — COMMENTED** [review]
哦对，还有一点，在math这里把论文的链接贴上吧

**Comment by jackalcooper** [comment]
https://github.com/Oneflow-Inc/oneflow/runs/2812294881

这个问题解决了吗，没解决不要加 automerge 啊，浪费CI资源

File: oneflow/python/nn/optimizer/cosine_annealing_lr.py [hunk 1]
```diff
@@ -55,26 +59,30 @@ class CosineScheduler(LrScheduler):
...
-
-    def get_lr(self):
-        if self.last_step < self.steps:
-            cos_decay = 0.5 * (1 + math.cos(math.pi * self.last_step / self.steps))
-            decay_factor = (1 - self.alpha) * cos_decay + self.alpha
-            return [base_lr * decay_factor for base_lr in self.base_lrs]
-        else:
-            return [base_lr * self.alpha for base_lr in self.base_lrs]
+
+def __init__(
```
Comment (wyg1997) [inline oneflow/python/nn/optimizer/cosine_annealing_lr.py:72]: 这里改错了

File: oneflow/python/nn/optimizer/cosine_annealing_lr.py [hunk 2]
```diff
@@ -55,13 +59,16 @@ class CosineScheduler(LrScheduler):
...
         for epoch in range(num_epoch):
             train(...)
-            cosine_scheduler.step()
+            cosine_annealing_lr.step()
 
+    .. _SGDR\: Stochastic Gradient Descent with Warm Restarts:
+        https://arxiv.org/abs/1608.03983
     """
 
+
```
Comment (wyg1997) [inline oneflow/python/nn/optimizer/cosine_annealing_lr.py:71]: 有两处多余的空行用flake8检察一下
